### DISCUSSION TITLE
From TWiki to Markdown: Words in PascalCase not converted to links

### DISCUSSION THREAD

**Discussion by oraculix** [jgm/pandoc#9941] [General]
Hi there,
while converting an old Wiki to Markdown, it occurred to me that all links in classic WikiLinkSyntax aren't converted to Markdown Links. I tried with markdown, markdown_mmd and gfm, all with the same negative results.

Environment(s):
- Fedora Linux 40, x64
- pandoc 3.1.3 (-server +lua)
- also pandoc 3.2 (+server +lua) from the Docker Registry

Example input:
```
This is a ClassicWikiLink.

This is an !EscapedWikiWord that must not be converted to a link.
```

Command line: `pandoc -f twiki -t markdown test.txt `

Expected output:
```
This is a [ClassicWikiLink](ClassicWikiLink).

This is an EscapedWikiWord that must not be converted to a link.
```

Example output:
```
This is a ClassicWikiLink.

This is an EscapedWikiWord that must not be converted to a link.
```

Interestingly, `!EscapedWikiWords` are respected as such, but why are links not created, then?
Before I open an issue, maybe something very simple escaped me? 
I mean, internal WikiLinks are one of the most basic features of a Wiki, so shouldn't this be implemented (above expected output would work for my case, going to Obsidian.md)?

Cheers,
Uwe

**Comment by jgm** [comment]
Indeed, automatic linking of CamlCase words doesn't seem to be implemented in the twiki reader. We could add it easily enough; please submit an issue.

**Reply by jgm** [reply]
Never mind, I've just fixed this.

**Reply by oraculix** [reply]
Thanks for the prompt answer, John. And, BTW, thanks for your work on the fantastic pandoc!
Issue / feature request #9944 is opened.

Kind regards,
Uwe

**Reply by oraculix** [reply]
> Never mind, I've just fixed this.

Oops, race condition occurred. :-D

**Comment by oraculix** [comment]
BTW: For anyone interested in the workarounds I use currently – particularly for several TWiki Macros, like attachments and highlighted code – see my `sed` overkill here:
https://codeberg.org/oraculix/TWiki2Obsidian
